src/seedsigner/views/tools_views.py

```diff
@@ -496,16 +496,8 @@ def run(self):
             return Destination(SeedMnemonicEntryView)
 
         elif button_data[selected_menu_num] == self.TYPE_ELECTRUM:
-            self.run_screen(
-                    WarningScreen,
-                    title="Electrum warning",
-                    status_headline=None,
-                    text=f"Some features disabled for Electrum seeds",
-                    show_back_button=False,
-            )
-            from seedsigner.views.seed_views import SeedMnemonicEntryView
-            self.controller.storage.init_pending_mnemonic(num_words=12, is_electrum=True)
-            return Destination(SeedMnemonicEntryView)
+            from seedsigner.views.seed_views import SeedElectrumWarningView
+            return Destination(SeedElectrumWarningView)
```

Comment:
I believe you meant to call SeedElectrumMnemonicStartView here, perhaps renamed it in the middle of the refactor

Reply:
This is what I get for being too dependent on VSCode to just magically get things right...

(...okay, and for not having enough FlowTest scenarios to catch things!)